canonical-web-and-design/canonical.com
# Add new cookie policy module

## Done

- Add new cookie policy module

## QA

- Check out this feature branch
- Run the site using the command ./run serve
- View the site locally in your web browser at: http://0.0.0.0:8002/
- Run through the following QA steps
- See the new cookie popup and try the settings
- Test the "Manage your tracker settings" link from the footer


## Issue / Card

Fixes #289
